docs/blog/_posts/2019-04-15-14th-dotty-milestone-release.md

```diff
@@ -0,0 +1,200 @@
+
+ - otherwise, if `f` is syntactically a stable identifier, and `new f` where `f` is interpreted as a type identifier is applicable to `args`, continue with `new f(args)`.
+
+For more information, please see the [documentation](http://dotty.epfl.ch/docs/reference/other-new-features/creator-applications.html).
+
+## Other changes
+
+Some of the other changes include:
+
+- `infer` method renamed to `the`, the semantics of which is now the same as that of the `the` method of Shapeless. Namely, the implicits are resolved more precisely – see this [gist](https://gist.github.com/milessabin/8833a1dbf7e8245b30f8) for an example in Shapeless, and the Dotty [documentation](http://dotty.epfl.ch/docs/reference/contextual/inferable-params.html#querying-implied-instances) for more details.
+- The syntax of quoting and splicing was changed. Now the quoting is expressed via `'{ ... }` and `'[...]` and splicing – via `${...}` and `$id`. Please see the [documentation](http://dotty.epfl.ch/docs/reference/other-new-features/principled-meta-programming.html) for more details on these features.
+
+# Let us know what you think!
+
+If you have questions or any sort of feedback, feel free to send us a message on our
+[Gitter channel](https://gitter.im/lampepfl/dotty). If you encounter a bug, please
+[open an issue on GitHub](https://github.com/lampepfl/dotty/issues/new).
+
+## Contributing
+
+Thank you to all the contributors who made this release possible!
+
+According to `git shortlog -sn --no-merges 0.13.0-RC1..0.14.0-RC1` these are:
+
+```
+   214  Martin Odersky
+   151  Nicolas Stucki
+    71  Liu Fengyun
+    53  Guillaume Martres
+    26  Olivier Blanvillain
+    10  Aleksander Boruch-Gruszecki
+     9  Aggelos Biboudis
+     6  Miles Sabin
+     4  Allan Renucci
+     4  Dale Wijnand
+     3  Anatolii Kmetiuk
+     2  Fengyun Liu
+     2  Alex Zolotko
+     1  gnp
+     1  tim-zh
+     1  Dmitry Petrashko
+     1  Dotty CI
+     1  Jasper Moeys
+     1  Jentsch
+     1  Jim Van Horn
+     1  Lionel Parreaux
+     1  Master-Killer
+     1  Olivier ROLAND
+     1  Robert Stoll
+     1  Seth Tisue
+     1  Tomasz Godzik
+     1  Victor
+```
+
+If you want to get your hands dirty and contribute to Dotty, now is a good time to get involved!
+Head to our [Getting Started page for new contributors](https://dotty.epfl.ch/docs/contributing/getting-started.html),
+and have a look at some of the [good first issues](https://github.com/lampepfl/dotty/issues?q=is%3Aissue+is%3Aopen+label%3Aexp%3Anovice).
+They make perfect entry points into hacking on the compiler.
+
+We are looking forward to having you join the team of contributors.
+
+## Library authors: Join our community build
+
+Dotty now has a set of widely-used community libraries that are built against every nightly Dotty
+snapshot. Currently this includes ScalaPB, algebra, scalatest, scopt and squants.
+Join our [community build](https://github.com/lampepfl/dotty-community-build)

```

Comment:
This URL is outdated, the community build is now in https://github.com/lampepfl/dotty/tree/master/community-build which lacks documentation unfortunately.